root/static/scripts/timeline.js

```diff
@@ -212,13 +212,26 @@ class TimelineViewModel {
 
         _.forEach(parts, function (part) {
             var match;
-            if (match = part.match(/^(-)?([rv])-?$/)) { // trailing - for backwards-compatibility
+
+            /** Please correct me if i'm misunderstanding this
+             * if(match = part.match(RegExp) {
+             *    // Do sth!!
+             * }
+             * 
+             * is equivalent to
+             * 
+             * if(part.match(RegExp)) {
+             *    match = part.match(RegExp);
+             *    // Do sth!!
+             * }
+             *  */ 
```

Comment:
Yes, but it is not good practice https://stackoverflow.com/questions/2576571/assign-variable-in-if-condition-statement-good-practice-or-not